container/cypress/e2e/test-app/compound/wc-compound-container.cy.js

```diff
@@ -1,26 +1,29 @@
 describe('Compound Container Tests', () => {
   describe('LuigiClient API - LuigiCompoundContainer', () => {
+    const container = '[data-test-id="luigi-client-api-test-compound-01"]';
```

Comment:
```suggestion
    const containerSelector = '[data-test-id="luigi-client-api-test-compound-01"]';
```

This name fits better I think